.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
Yup - following arbitrary style rules is indeed annoying. That is the price you have to pay to have consistent code across many people. 

> The way I see it, there merely happen to be no other unused kwargs. But if there were, we'd just disable the cop there too.

Again - as of right now this is not the correct approach to take. I would love to get this PR merged but in order to do so you'll have to follow one of the two paths I requested earlier.